pom.xml

```diff
@@ -19,6 +19,7 @@
         <javafaker.version>1.0.2</javafaker.version>
         <projectlombok.version>1.18.24</projectlombok.version>
         <configuration.version>1.10</configuration.version>
+        <junit.version>4.12</junit.version>
```

Comment:
Is there any reason to add junit as a dependency? Testng and hamcrest combined probably already cover all of your assert needs

Reply:
I wasn't aware of that...
In that case I will remove junit dependency.